README.md

```diff
@@ -1,4 +1,38 @@
 ![](https://github.com/genesys/mobiledx-samples-ios/blob/master/GenesysMessengerSample/1280x720-PR-Meta-Image.jpeg)
 
+# Overview
+
+This repository contains sample iOS application demonstrating the use of the GC Mobile Messenger SDK. 
+
+## Getting Started
+To get started with these samples, clone this repository to your local machine:
+
+```bash
+git clone git@github.com:MyPureCloud/mobiledx-samples-ios.git
+```
+
+## Prerequisites
+- iOS 11 and above
```

Comment:
Based on the fact that we already raised the minimum iOS version, maybe open a ticket to update both the documentation on the developer center & this line on the readme?